### ISSUE TITLE
Simple icons

### ISSUE THREAD

**Issue by Victor-Bo** [keeweb/keeweb#1695]
**Is your feature request related to a problem? Please describe.**
We can kinda say it's related to a problem, I'd like to get icons easily

**Describe the solution you'd like**
Instead of or below the icons that are here I'd like to have also a list of the simple icons: https://simpleicons.org/

**Describe alternatives you've considered**
Icons can be accessed using the following url :
https://simpleicons.org/icons/BRANDNAME.svg
for example
https://simpleicons.org/icons/keepassxc.svg

You could: 
- make a list of the icons below the icons
- make a field or some place where we can input the brand name and fetch automatically the icon

**Additional context**

![Sans titre](https://user-images.githubusercontent.com/39555268/106160824-8b914f00-6186-11eb-8b64-fdcaba3fe084.png)
(of course there would be much more icons here)

**Comment by antelle** [comment]
Well, there are several things here:
1. not sure if SimpleIcons provide API with CORS so that it's available for browsers
2. icons will be saved in PNG, not SVG, so the file size will grow, unlike icons that we have that are just saved as a short ID
3. because of 2., colors won't be changed

**Comment by Victor-Bo** [comment]
1. Yes it does
![image](https://user-images.githubusercontent.com/39555268/106193103-5ba97200-61ad-11eb-9044-d94c79373b65.png)
2. I didn't understand what you mean so I guess you're right ¯\_(ツ)_/¯
3. All SVGs are provided in black so I think you could manage to change the colors but again I guess you have good reasons not to be able to do it

**Comment by antelle** [comment]
The format that we're using (kdbx) stores images in PNG, not SVG. If we store them in SVG, our format will not be compatible to other clients, for example, you won't be able to open it with KeePassXC, that's why we need to save it as PNG.

**Comment by Victor-Bo** [comment]
Ah ok I understand, why will it be different than the icons you already have?

**Comment by antelle** [comment]
KDBX format has a pre-defined set of icons that appear in all KeePass clients. These icons are identified with a number from 1 to 68, the actual graphics is different across all forks, but most of icons are similar and express the same concept. For example, a lock icon will look like lock everywhere, while obscure things like a certificate sign (or was it a star?) are somewhat different.

**Comment by Victor-Bo** [comment]
I ok I understand, I thought that was specific to KeeWeb. I let you see then!

**Comment by antelle** [comment]
No, the format is shared, this makes it possible to use any clients and switch between them a often as you want.

My thoughts on the feature, it’s helpful because now you can also attach a custom icon, so if there’s anyone who would like to contribute, it would be accepted. There are some things to solve here though.

**Comment by pwnoobz** [comment]
> 

I was reading this topic because I also found myself looking for other icons to organise the .kdbx file in a better way. Seems you can add icons but can you delete them after? I accidentally added a big filesize icon.

**Comment by antelle** [comment]
If you delete an entry with this icon, it will disappear. Don’t forget to delete it drom trash also. If the entry is needed, delete a state from history when this icon was in use.

**Comment by pwnoobz** [comment]
Thank you for the hint!

> If you delete an entry with this icon, it will disappear. Don’t forget to delete it drom trash also. If the entry is needed, delete a state from history when this icon was in use.
